fleet/lib/nexus-pg/overlays/nexus-pg-cluster.yaml

```diff
@@ -0,0 +1,48 @@
+---
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
+metadata:
+  name: nexus-pg
+spec:
+  imageName: ghcr.io/cloudnative-pg/postgresql:15
+
+  instances: 1
+
+  bootstrap:
```

Comment:
this is not needed, this can be setup manually just once and then will be saved in the backup